common/religion/religions/secular_group.txt

```diff
@@ -24,6 +24,20 @@
 	doctrine = doctrine_deviancy_shunned
 	doctrine = doctrine_witchcraft_crime
 
+	#Magic
+	doctrine = doctrine_light_magic_ignored
+	doctrine = doctrine_shadow_magic_crime
+	doctrine = doctrine_disorder_magic_crime
+	doctrine = doctrine_order_magic_approved
+	doctrine = doctrine_life_magic_ignored
+	doctrine = doctrine_death_magic_crime
+	doctrine = doctrine_elemental_spirit_magic_ignored
+	doctrine = doctrine_elemental_decay_magic_ignored
+	doctrine = doctrine_elemental_fire_magic_accepted
+	doctrine = doctrine_elemental_water_magic_accepted
+	doctrine = doctrine_elemental_air_magic_accepted
+	doctrine = doctrine_elemental_earth_magic_accepted
```

Comment:
Light, Order, Spirit Accepted.
Gnomes, goblins don't mind all kinds of magic.

Reply:
Done?

Reply:
> Done?

Order Accepted instead of Approved. They don't worship magic.
Also, maybe it will be a thing of Secular? They can't have Approved?

Reply:
Ugh
That's will hurt them a LOT, but fine, I guess

Reply:
> Ugh
> That's will hurt them a LOT, but fine, I guess

Mmmkay, keep it then.